README.rst

```diff
@@ -31,6 +31,8 @@ moment (if you follow `Compilation of Octopus using Spack`_):
 
 |spack-develop-octopus-develop| Spack develop version, develop version of Octopus
 
+|diff-with-upstream| A check to see if this repository is in sync with the upstream spack repository.
```

Comment:
```suggestion
|diff-with-upstream| A check to see if our `octopus/package.py` file is in sync with the upstream spack repository.
```

Reply:
Adding the link to the actual  file (or files) would be cool (but  not strictly needed).